config/recurring.yml

```diff
@@ -15,3 +15,6 @@ production:
   repopulate_location_and_observation_center_lat_lng:
     class: UpdateBoxAreaAndCenterColumnsJob
     schedule: every day at midnight
+  discard_stale_jobs:
+    class: DiscardStaleJobsJob
+    schedule: every day at midnight
```

Comment:
Should this also have an `argument`?

Reply:
What are you thinking?  This seems reasonable to me.

Reply:
If discard_date is an arg you will want to pass, then yes, otherwise i guess not?